### DISCUSSION TITLE
My page is accessible at https://USER.github.io/index.html, but not at https://USER.github.io/

### DISCUSSION THREAD

**Discussion by Scoty** [community/community#22874] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, all,</p>
<p>Can someone tell me why my page opens just fine at: <a href="https://scoty.github.io/index.html" rel="nofollow noopener">https://scoty.github.io/index.html</a> , but at <a href="https://scoty.github.io/" rel="nofollow noopener">https://scoty.github.io/</a> I get the default GitHubPage’s 404 page and not the custom 404 page that I have defined(via 404.html file at the root) ?</p>

**Comment by enderahmetyurt** [comment]
<p>I checked and it’s working fine. What’s the problem?</p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/scoty">@scoty</a>,</p>
<p>I am also able to pull up your site at <a href="https://scoty.github.io/" rel="nofollow noopener">https://scoty.github.io/</a> . Are you still experiencing this problem? If so, what browser are you using?</p>

**Comment by Scoty** [comment, marked as answer]
<p>Everything is working now.</p>
